{{cookiecutter.plugin_name}}/{{cookiecutter.module_name}}/_tests/test_reader.py

```diff
@@ -26,5 +29,7 @@ def test_reader(tmp_path):
 
 
 def test_get_reader_pass():
+    # make_napari_viewer block plugin discovery, so we must register it manually
+    napari_plugin_manager.register({{cookiecutter.module_name}}, name="{{cookiecutter.plugin_name}}")
```

Comment:
same here... not necessary, since the plugin function is being called directly.  (also, you'd need to add the `napari_plugin_manager` fixture to the signature)

side note: if I were testing a plugin, I'd try to "leave napari out of it" as much as possible, and test my functions directly... adding only one or two tests that make sure the plugin is actually being discovered and making it through the necessary napari code as expected.